src/parachain/tokens.ts

```diff
@@ -103,8 +107,12 @@ export class DefaultTokensAPI extends DefaultTransactionAPI implements TokensAPI
 
     async transfer<U extends CurrencyUnit>(destination: string, amount: MonetaryAmount<Currency<U>, U>): Promise<void> {
         const amountSmallDenomination = this.api.createType("Balance", amount.toString());
```

Comment:
Naming convention here. I think in some other places you named this `amountAtomicUnit`. Good to stick with one name for the same thing.